laravel-scout-extended/.env.example

```diff
@@ -0,0 +1,42 @@
+APP_NAME=Laravel
+APP_ENV=local
+APP_KEY=base64:1GRujYRd/dLHOFiPAfKSI4aDBt8enncJlszMlfujgRs=
```

Comment:
You should NOT make this public. Run on composer install this command: `php artisan key:generate`.

Reply:
👍